Resources/Prototypes/Roles/Jobs/CentComm/emergencyresponseteam.yml

```
@@ -207,6 +207,7 @@
     - type: CanEnchant #GoobStation
   - !type:AddImplantSpecial
      implants: [ MindShieldImplant, FreedomImplant, BluespaceLifelineImplant, RadioImplantCentcomm, NutrimentPumpImplant, StypticStimulatorImplant, DeathRattleImplantCentcomm ]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)